src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala

```diff
@@ -80,11 +80,15 @@ object ContainerSerializer {
         builder.setMesos(MesosAppCSerializer.toMesos(ma))
     }
 
-    container.volumes.foreach {
-      case _: PersistentVolume => // PersistentVolumes are handled differently
-      case ev: ExternalVolume => ExternalVolumes.build(builder, ev) // this also adds the volume
-      case dv: DockerVolume => builder.addVolumes(VolumeSerializer.toMesos(dv))
-      case _: SecretVolume => // SecretVolumes are handled differently
+    container.volumes.foreach { volumeWithMount =>
+      val volume = volumeWithMount.volume
+      val mount = volumeWithMount.mount
+      volume match {
+        case _: PersistentVolume => // PersistentVolumes are handled differently
+        case ev: ExternalVolume => ExternalVolumes.build(builder, ev, mount) // this also adds the volume
+        case dv: HostVolume => builder.addVolumes(VolumeSerializer.toMesos(dv, mount))
+        case _: SecretVolume => // SecretVolumes are handled differently
```

Comment:
Do you think it's a good idea to return something different than Unit?

Reply:
Could you please tell me what is better to return here and why?